Interface.C

```diff
@@ -293,25 +310,16 @@ void preciceAdapter::Interface::configureMesh(const fvMesh& mesh, const std::str
                     {
                         for (uint nodeIndex = 0; nodeIndex < nodesPerTria; nodeIndex++)
                         {
-                            for (uint xyz = 0; xyz < componentsPerNode; xyz++)
-                                triCoords[coordIndex++] = pointCoords[faceTri[triIndex][nodeIndex]][xyz];
+                            triVertIDs.push_back(verticesMap.at(std::make_tuple(pointCoords[faceTri[triIndex][nodeIndex]][0], pointCoords[faceTri[triIndex][nodeIndex]][1], pointCoords[faceTri[triIndex][nodeIndex]][2])));
                         }
                     }
                 }
 
-                //Array to store the IDs we get from preCICE
-                int triVertIDs[faceField.size() * (triaPerQuad * nodesPerTria)];
-                /*
-                //Get preCICE IDs
-                precice_.getMeshVertexIDsFromPositions(meshID_, faceField.size() * (triaPerQuad * nodesPerTria), triCoords, triVertIDs);
-
-                DEBUG(adapterInfo("Number of triangles: " + std::to_string(faceField.size() * triaPerQuad)));
+                DEBUG(adapterInfo("Number of faces * triaPerQuad: " + std::to_string(faceField.size() * triaPerQuad)));
+                DEBUG(adapterInfo("Number of triangles: " + std::to_string(triVertIDs.size())));
```

Comment:
Do you want to keep the debug log?
If the answer is yes, then the bottom one needs to be corrected. Its either `"Number of triangle ids:"` or `triVertIDs.size()/3`.